CMakeLists.txt

```diff
@@ -1,63 +1,46 @@
 PROJECT(pblib)
 
-cmake_minimum_required(VERSION 2.8)
```

Comment:
This directive should _always_ be first thing in a CMakeLists.txt, see [documentation](https://cmake.org/cmake/help/latest/command/cmake_minimum_required.html)